src/contracts/atlas/Escrow.sol

```diff
@@ -392,7 +392,8 @@ abstract contract Escrow is AtlETH {
         if (!_result.canExecute()) return 0;
 
         uint256 _gasLimit;
-        (, _gasLimit) = _validateSolverOpGas(dConfig, solverOp, _gasWaterMark);
+        (_result, _gasLimit) = _validateSolverOpGas(dConfig, solverOp, _gasWaterMark);
+        _result |= _validateSolverOpDeadline(solverOp, dConfig);
 
         if (dConfig.callConfig.allowsTrustedOpHash()) {
             if (!_handleAltOpHash(userOp, solverOp)) {
```

Comment:
`_result` isn't check after this, it probably makes sense to move this above the `_result.canExecute()` check

Reply:
Agreed

Reply:
Done here https://github.com/FastLane-Labs/atlas/pull/310/commits/d0d4f1a456565e33c3147659b461c3a3beb9f274